src/xtgeo/xyz/_xyz_roxapi.py

```diff
@@ -27,7 +27,7 @@ def _check_stypes_names_category(roxutils, stype, name, category, export=False):
 
     if stype not in valid_stypes:
         raise ValueError(
-            f"Given stype {stype} is not supported, legal stypes are: {valid_stypes}"
+            f"Invalid stype {stype} is not supported, legal stypes are: {valid_stypes}"
```

Comment:
The error message seems to indicate that `stype` is both invalid and not supported. Might be confusing.

Reply:
Will rephrase